app/javascript/components/profile-dropdowns.vue

```diff
@@ -60,5 +76,31 @@ export default {
       selectedOrganizationId: state => state.profile.selectedOrganizationId,
     }),
   },
+  methods: {
+    onMonthClick(index) {
+      this.selectedMonth = index;
+    },
+    onOrganizationClick(index) {
+      this.selectedOrganization = index;
+    },
+    onTeamClick(index) {
+      debugger;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using 'DebuggerStatement' is not allowed  no-restricted-syntax